sdk/keyvault/azure-security-keyvault-keys/pom.xml

```diff
@@ -95,7 +95,14 @@
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-test</artifactId>
-        <version>1.1.0</version> <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
+        <version>1.2.0-beta.1</version> <!-- {x-version-update;unrelease_com.azure:azure-core-test;current} -->
```

Comment:
Same comment as above, the tag is incorrect and should be x-version-update;unreleased_com.azure:azure-core-test;dependency